lib/cmd/fh3/admin/users/create.js

```diff
@@ -0,0 +1,82 @@
+/* globals i18n */
+var common = require("../../../../common");
+var fhreq = require("../../../../utils/request");
+
+module.exports = {
+  'desc' : i18n._('Create a RHMAP User'),
+  'examples' :
+    [{
+      cmd : 'fhc admin users create --username=<username> --password=<password> --email=<email> --name=<name> --roles=<roles> --authpolicies=<authpolicies> --invite=<true|false> --teams=<teams> --groups=<groups>',
+      desc : i18n._('Create a RHMAP user with <username>, <password>, <email>, <name>, <roles>, <authpolicies>, <invitation>, <teamIds>, <groups>')
+    }],
+  'demand' : ['username','email'],
+  'alias' : {
+    'username' : 'u',
+    'password' : 'p',
+    'email' : 'e',
+    'name' : 'n',
+    'roles' : 'r',
+    'authpolicies' : 'a',
+    'invite' : 'i',
+    'teams' : 't',
+    'groups' : 'g',
+    'json' : 'j',
+    0 : 'username',
+    1 : 'email',
+    2 : 'password',
+    3 : 'name' ,
+    4 : 'roles',
+    5 : 'authpolicies',
+    6 : 'invite',
+    7 : 'teams',
+    8 : 'groups'
+  },
+  'describe' : {
+    'username' : i18n._("Username of the new user."),
+    'email' : i18n._("Email of the new user. It will be used to send the invite email."),
+    'password' : i18n._("Password for this user login into RHMAP"),
+    'name' : i18n._("Name of the new user"),
+    'roles' : i18n._("Roles permissions for the user. (E.g 'sub')"),
+    'authpolicies' : i18n._("Unique 24 character GUID of Auth Policies. Run '$fhc admin policies list' to check it."),
+    'invite' : i18n._("Boolean to define if is to  send or not an email invitation for this new user"),
+    'teams' : i18n._("Unique 24 character GUID of the teams which the new user will be a member"),
+    'groups' : i18n._("Unique 24 character GUID of the groups which the new user will be a member"),
+    'json' : i18n._("Output in json format")
```

Comment:
inconsistency. In most of the other commands we use `--json`

Reply:
Yeah. not sure why I have added that comment. Sorry.

Reply:
@wtrocki No Worry .. really tks for your review and all help so far !!!